code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -0,0 +1,412 @@
+		return ..()
+	if(!cached_reagent_list)
+		return ..()
+	for(var/datum/reagent/reagent as anything in cached_reagent_list)
+		if(!reagent)
+			continue
+		reagent.creation_purity *= 1.25
+	cached_reagent_list = null
+
+////Multiver
+//Inverse
+//Reaction product when between 0.2 and 0.35 purity.
+/datum/reagent/inverse/healing/monover
+	name = "Monover"
+	description = "A toxin treating reagent, that only is effective if it's the only reagent present in the patient."
+	ph = 0.5
+	addiction_types = list(/datum/addiction/medicine = 3.5)
+
+//Heals toxins if it's the only thing present - kinda the oposite of multiver! Maybe that's why it's inverse!
+/datum/reagent/inverse/healing/monover/on_mob_life(mob/living/carbon/owner, delta_time, times_fired)
+	if(owner.reagents.reagent_list > 1)
+		owner.adjustOrganLoss(ORGAN_SLOT_LUNGS, 0.5 * delta_time) //Hey! It's everyone's favourite drawback from multiver!
+		return ..()
+	owner.adjustToxLoss(-2 * REM * creation_purity * delta_time, 0)
+	..()
+	return TRUE
+
+///Can bring a corpse back to life temporarily (if heart is intact)
+///Makes wounds bleed more, if it brought someone back, they take additional brute and heart damage
+///They can't die during this, but if they're past crit then take increasing stamina damage
+///If they're past fullcrit, their movement is slowed by half
+///If they OD, their heart explodes (if they were brought back from the dead)
+/datum/reagent/inverse/penthrite
+	name = "Nooartrium"
+	description = "A reagent that is known to stimulate the heart in a dead patient, temporarily bringing back recently dead patients at great cost to their heart."
+	ph = 14
+	metabolization_rate = 1 * REM
+	addiction_types = list(/datum/addiction/medicine = 12)
+	overdose_threshold = 20
+	///If we brought someone back from the dead
+	var/back_from_the_dead
+
+/datum/reagent/inverse/penthrite/on_mob_dead(mob/living/carbon/owner)
+	var/obj/item/organ/heart/heart = owner.getorganslot(ORGAN_SLOT_HEART)
+	if(!heart || heart.organ_flags & ORGAN_FAILING)
+		return ..()
+	ADD_TRAIT(owner, TRAIT_STABLEHEART, type)
+	ADD_TRAIT(owner, TRAIT_NOHARDCRIT, type)
+	ADD_TRAIT(owner, TRAIT_NOSOFTCRIT, type)
+	ADD_TRAIT(owner, TRAIT_NOCRITDAMAGE, type)
+	owner.stat = CONSCIOUS
+	back_from_the_dead = TRUE
+	owner.emote("gasp")
+
+/datum/reagent/inverse/penthrite/on_mob_life(mob/living/carbon/owner, delta_time, times_fired)
+	owner.playsound_local(owner, 'sound/health/slowbeat.ogg', 40)
+	if(back_from_the_dead)
+		owner.adjustBruteLoss(5 * (1-creation_purity) * delta_time)
+		owner.adjustOrganLoss(ORGAN_SLOT_HEART, 2.5 * (1-creation_purity) * delta_time)
+	for(var/datum/wound/iter_wound as anything in owner.all_wounds)
+		iter_wound.blood_flow += (1-creation_purity)
+	if(owner.health < HEALTH_THRESHOLD_CRIT)
+		owner.add_movespeed_modifier(/datum/movespeed_modifier/reagent/nooartrium)
+	if(owner.health < HEALTH_THRESHOLD_FULLCRIT)
+		owner.add_actionspeed_modifier(/datum/actionspeed_modifier/nooartium)
```

Comment:
imagine needing to worry about any of these effects, couldn't be me

this post brought to you by liverless gang

Reply:
also, "nooartrium" is misspelled in that actionspeed_modifier typepath

Reply:
Fixed the nooartrium. I thought on_mob_life was independant of liver?

Reply:
Oh, I just need to set self_consuming = TRUE for it to work without a liver.